Be/unit1 wrap up (#118)
Ada-Developers-Academy/core-unit-1: unit1-wrap-up/activity-mind-map.md

> # Unit 1 - Reflection Activity

Reviewer: Could we conform this to:

```suggestion
# Activity: Mind Map
```

? I'm really hoping for file names, lesson names, what's on the config, and what shows up on Learn, to all be aligned (so that we don't lose files as easily as we did in textbook-curriculum)

Author: incorporated.

> 

Reviewer: Nitpicking: I know this is goofy, but I'm suggesting a template header.

```suggestion

## Goals

```

Author: incorporated.

> Wow! We've learned so much in the last 5 weeks. Let's take time to reflect on our learning and make connections.

Reviewer: Because it'd be great to not tie our curriculum to time passing, could we divorce it from "five weeks"?

```suggestion
Wow! We've learned so much in this unit. Let's take time to reflect on our learning and make connections.
```

> 

Reviewer: Nitpicking: Adding more of the headers from the activity template

```suggestion

Activity Instructions

```

We will use a technique called a mind map. A mind map is a diagram that can represent tasks, ideas, concepts, etc. and the connections between them. In our case, we can include anything from our learnings from Unit 1 -- whether it be python programming concepts, study skills, learnings about how to be in community, or anything else.

These are a few tools our team recommends: 
* [Google Slide](https://www.google.com/slides/about/)
* [Mind Meister](https://www.mindmeister.com/)

> and here is a [more exhaustive list](https://zapier.com/blog/best-mind-mapping-software/) of mind mapping software.

Reviewer: Why not just pick a few from this list? A number of these aren't free

Author: I tried to pick some, and I didn't love anything

> We will use a technique called a mind map. A mind map is a diagram that can represent tasks, ideas, concepts, etc. and the connections between them. In our case, we can include anything from our learnings from Unit 1 -- whether it be python programming concepts, study skills, learnings about how to be in community, or anything else.
> 
> These are a few tools our team recommends: 
> * [Google Slide](https://www.google.com/slides/about/)
> * [Mind Meister](https://www.mindmeister.com/)
> 
> and here is a [more exhaustive list](https://zapier.com/blog/best-mind-mapping-software/) of mind mapping software.
> 
> We encourage you to pick a tool and go for it -- no censorship needed.

Reviewer: Recommending for this be reworded so the "we" is about us learners, without the instructor/team divide.

Also recommending "we will" -> "let's" since it's about to happen.

- Capitalization suggestions
- Framing the directions as activity instructions instead of as suggestions
```suggestion
Let's create a mind map. A mind map is a diagram that can represent tasks, ideas, and concepts, and the connections between them.

In our case, let's include any topic from our learnings from this unit. We should consider:

- Python programming concepts
- Study skills
- Learnings about how to be in community
- Anything else!

We can make our mind map using any of these tools:

- [Google Slide](https://www.google.com/slides/about/)
- [Mind Meister](https://www.mindmeister.com/)
- [Software from this list of mind mapping software](https://zapier.com/blog/best-mind-mapping-software/)
```

Author: I didn't see this before I started making changes, so I incorporated it, but didn't straight up commit the suggestion.

## Examples

To learn more about the benefits of mind mapping, read this mind map!

![how-to-mind-map](../assets/unit-1-reflection-mind-map.jpg)
*Fig: Mind-map map [(source).](https://www.mindmeister.com/blog/wp-content/uploads/2015/01/MindMapping_mindmap_handdrawn.png)*

Here's a mind map an individual shared of their life which demonstrates the power of making connections across different areas.

![how-to-mind-map](../assets/unit-1-reflection-my-life-mind-map.png)
*Fig: Life mind-map [(source).](http://emmacookbehaviourandenvironment.blogspot.com/)*

## Activity Instructions

Let's create a mind map. A mind map is a diagram that can represent tasks, ideas, and concepts, and the connections between them.

In our case, let's include any topic from our learnings from this unit. We should consider:
- Python programming concepts
- Study skills
- Learnings about how to be in community
- Anything else!

We can make our mind map using any of these tools:
- [Google Slide](https://www.google.com/slides/about/)
- [Mind Meister](https://www.mindmeister.com/)
- [Software from this list of mind mapping software](https://zapier.com/blog/best-mind-mapping-software/)

> We will spend approximately 40 minutes in small groups to work together and include all the wisdom from our classmates. We will come back together as a large group to share insights.

Reviewer: Can we change the language here to:

```suggestion
Spend approximately 40 minutes in small groups to work together and include all the wisdom from our classmates. 

Then, come back together as a large group to share insights.
```

Personally, I think that having example questions that the groups should share with the large group might be helpful. (For example, once it gets to large group sharing, the groups might be like "uhhh what do we share?"). For example, will it be a presentation, or just an open discussion?

If it's a presentation, I might include presentation directions (each group will present for 5 minutes each) and maybe:

```
When other groups are presenting, ask yourselves:

- What topics did this group and your group have in common?
- When comparing similar topics, what connections were similar?
- What connections were different?
- What can you learn from them?
```

If it's just an open discussion, I'd propose giving them the first opening question,

"We'll start the large group discussion with each group answering the question: ..."
